packages/scoutgame/src/points/refreshBuilderLevels.ts

```diff
@@ -26,5 +26,20 @@ export async function refreshBuilderLevels({ season = getCurrentSeasonStart() }:
       }
     });
   }
-  log.info(`Refreshed builder levels for season ${season} with ${levels.length} builders`);
+
+  const levelZeroBuilders = await prisma.userSeasonStats.updateMany({
```

Comment:
in what situation do we need to set builders back to level 0?

Reply:
Thanks, this is a one off situation and should actually be in the script